python/cog/schema.py

```diff
@@ -38,18 +38,18 @@ def default_events(cls) -> t.List["WebhookEvent"]:
         return [cls.START, cls.OUTPUT, cls.LOGS, cls.COMPLETED]
 
 
-class PredictionBaseModel(pydantic.BaseModel, extra=pydantic.Extra.allow):
+class PredictionBaseModel(pydantic.BaseModel, extra="allow"):
```

Comment:
Does Pydantic 1 support string literal? Or should we conditionalize?

```suggestion
class PredictionBaseModel(pydantic.BaseModel, extra="allow" if PYDANTIC_V2 else pydantic.Extra.allow):
```

Reply:
yup: https://docs.pydantic.dev/1.10/usage/model_config/#:~:text=Accepts%20the%20string%20values%20of%20%27ignore%27%2C%20%27allow%27%2C%20or%20%27forbid%27